Starwar/Assets/Scripts/Player Control/PlayerUI.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+
+using UnityEngine;
+using UnityEngine.UI;
+
+
+public class PlayerUI: MonoBehaviour
+{
+    GameObject aircraft;
+
+    GameObject healthText;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        aircraft = GameObject.Find("Aircraft");
```

Comment:
avoid use Find("objectName") or FindWithTag("TagName"). Very inefficient.
use a Public GameObject aircraft; and then set the aircraft in unity inspector.